### DISCUSSION TITLE
Github 404 Error

### DISCUSSION THREAD

**Discussion by ShedrachMorris** [vercel/community#7204] [Help]
### Summary

my html code works fine when i deploy locally, but i get a 404 page not found error when i deploy in my github repository. please what do you think is the problem.

[~](http://127.0.0.1:5500/) / [index.html](http://127.0.0.1:5500/index.html)
404
This page could not be found.

### Steps to Reproduce

[~](http://127.0.0.1:5500/) / [index.html](http://127.0.0.1:5500/index.html)
404
This page could not be found.

**Comment by amyegan** [comment]
Hey @ShedrachMorris. I've shared some tips for debugging 404 errors over in https://github.com/vercel/community/discussions/6980#discussioncomment-9705033

If you can share a public [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), someone here might be able to recognize what's going wrong in your case.

[a comment by an automated account is not shown]
